oeps/oep-0028-content-theming-in-react.rst

```diff
@@ -0,0 +1,206 @@
+==================================
+OEP-0028: Content Theming in React
+==================================
+
+.. This OEP template is based on Nygard's Architecture Decision Records.
+
++-----------------+----------------------------------------------------------------+
+| OEP             | :doc:`OEP-0028 </oeps/oep-0028-arch-content-theming-react>`    |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
++-----------------+----------------------------------------------------------------+
+| Title           | Content Theming in React                                       |
++-----------------+----------------------------------------------------------------+
+| Last Modified   | 2018-08-17                                                     |
++-----------------+----------------------------------------------------------------+
+| Authors         | Braden MacDonald                                               |
```

Comment:
?? I think this should be your name @taranjeet

Reply:
You can put both ;)

Reply:
Sure @bradenmacdonald :) Will do that.